recipes-support/provisioning/esec-provisioning.bb

```diff
@@ -0,0 +1,19 @@
+# Copyright (C) 2021 Maik Otto <m.otto@phytec.de>
+# Released under the MIT license (see COPYING.MIT for the terms)
+
+SUMMARY = "Provisioning Tools for ESEC IOTDM Plattform"
+LICENSE = "MIT"
+LIC_FILES_CHKSUM = "file://${COMMON_LICENSE_DIR}/MIT;md5=0835ade698e0bcf8506ecda2f7b4f302"
+
+FILESEXTRAPATHS_prepend := "${THISDIR}/files:"
+SRC_URI = " \
+    file://test-provision-tpm.sh \
+"
+
+S = "${WORKDIR}"
+
+do_install() {
+    install -m 0500 test-provision-tpm.sh ${D}/home
```

Comment:
You have to install `/home` as well, because otherwise this directory does not exist in the build folder for this package:
```
install -d ${D}/home
```

Reply:
Is there a specific reason why you install the script in `/home` rather than a common path like `/usr/bin`? If you want to make this only available for the root user, how about installing it in `/root`?

Reply:
changed to /usr/bin